data/uking_functions.csv

```diff
@@ -73409,13 +73409,13 @@
 0x0000007100d30d74,ActorInfoData::isAmiiboDrop,92,_ZNK4ksys3act8InfoData12hasDropEntryEPKcS3_
 0x0000007100d30dd0,ActorInfoData::getActorInstSize,20,_ZNK4ksys3act8InfoData11getInstSizeEPKc
 0x0000007100d30de4,ActorInfoData::getBugMask,20,_ZNK4ksys3act8InfoData10getBugMaskEPKc
-0x0000007100d30df8,sub_7100D30DF8,64,
-0x0000007100d30e38,sub_7100D30E38,408,
-0x0000007100d30fd0,sub_7100D30FD0,408,
+0x0000007100d30df8,sub_7100D30DF8,64,_ZN4ksys3act8InfoData21InvalidLifeConditions28containsCurrentTimeOrWeatherEPN4sead7Vector3IfEE
```

Comment:
These will need to be updated after you const-qualify the member functions (see the other review comment).